src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -243,6 +243,11 @@
   <ItemGroup Condition="'$(TargetGroup)' != 'netcoreapp'">
     <Compile Include="System\StringSplitExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
```

Comment:
Not sure if uapaot is also needed here. Probably yes till a new uapaot package is ingested.

Reply:
You can add the APIs to ApiCompatBaseline.uapaot.txt temporarily until the updated uapaot package is ingested.